PowerEditor/src/WinControls/TabBar/TabBar.h

```diff
@@ -53,7 +53,7 @@ constexpr int g_TabHeight = 22;
 constexpr int g_TabHeightLarge = 25;
 constexpr int g_TabWidth = 45;
 constexpr int g_TabWidthCloseBtn = 60;
-constexpr int g_TabCloseBtnSize = 11;
+constexpr int g_TabCloseBtnSize = 16;

```

Comment:
The left icons are the current old one, and the right ones are resized of current PR:

![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/90293/122619ef-588c-4ce8-9363-e713de552181)

Can't we keep the size 11?

Reply:
> Can't we keep the size 11?

Is there any possible way to keep 11 for the Standard icons and have the code do 16 when dark mode is selected? 

That would be the best for both scenarios if that is possible.

Reply:
@WildByDesign 
Yes. I think.
Please check the above request - I have just changed it.

Reply:
Thank you. I will do this but I away from my main system right now. I will test this out later in the day and less you know.